### ISSUE TITLE
rocm-opencl-sdk & rocm-hip-libraries missing from AUR

### ISSUE THREAD

**Issue by Mushoz** [rocm-arch/rocm-arch#932]
I initially started a discussion about this here: https://github.com/rocm-arch/rocm-arch/discussions/931

But since this seems to prevent me from running tensorflow-rocm, I am now reporting it as a bug rather than question.

As mentioned in the discussion, these 2 packages now return a 404 page not found on the AUR, making it unable to be installed via conventional ways such as via AUR helpers such as paru or yay.

**Comment by Mushoz** [comment]
rocm-language-runtime also seems to be missing

**Comment by tpkessler** [comment]
Thank you very much for bringing this to my attention. I submitted a merge request of all meta packages into a single package base a while back. After it was merged I forgot to update the new meta package. This is fixed now. However, I recommend to wait to little longer until you install / update ROCm. I'm currently updating all packages to 5.4.0. This time, it takes longer than usual as all packages are completely revised to make them ready for the official repo.
